### ISSUE TITLE
Empty continuation prompt breaks text placement near bottom of window 

### ISSUE THREAD

**Issue by lavirlifiliol** [PowerShell/PSReadLine#2723]
## Environment

```none
PS version: 7.1.3
PSReadline version: 2.2.0-beta3
os: 10.0.19041.320 (WinBuild.160101.0800)
PS file version: 7.1.3.0
HostName: ConsoleHost (Windows Terminal)
BufferWidth: 145
BufferHeight: 38
```

## Steps to reproduce

Run `Set-PSReadLineOption -ContinuationPrompt ''`, press enter until you reach the bottom of the terminal window, then press shift enter a number of times (alternatively, open a string literal). Then start typing regular text.

## Expected behavior
The text appears at the bottom of the window, as it does with a non empty prompt.
![image](https://user-images.githubusercontent.com/27435982/128270414-d1024903-c538-41ad-99c5-615b5f434805.png)

## Actual behavior
The text appears at a completely wrong place, but the cursor position remains consistent
![image](https://user-images.githubusercontent.com/27435982/128270490-fe1934b7-6687-49f5-9c7b-6abc21349385.png)

**Comment by rkeithhill** [comment]
I see the same issue:

![image](https://user-images.githubusercontent.com/5177512/128391851-a567fdd6-026f-4a62-a059-7147eba3c725.png)

**Comment by ghost** [comment]
:tada: This issue was addressed in 2875, which has now been successfully released in [`v2.2.0-beta4`](https://github.com/PowerShell/PSReadLine/releases/tag/v2.2.0-beta4). :tada:
